docroot/themes/humsci/humsci_basic/src/js/main-nav/nested-toggler.js

```diff
@@ -10,7 +10,7 @@ if (togglers) {
     const togglerID = toggler.getAttribute('id');
     const togglerContent = document.querySelector(`[aria-labelledby="${togglerID}"]`);
     const togglerParent = toggler.parentNode;
-    const subnavShouldBeExpanded = togglerParent.classList.contains('su-main-nav__item--active-trail') ? true : false;
+    const subnavShouldBeExpanded = togglerParent.classList.contains('hb-main-nav__item--active-trail') ? true : false;
```

Comment:
could you rename this to something a little bit clearer? I was thinking `subnavIsExpanded`

Reply:
or maybe: `parentIsExpanded`